code/drasil-lang/lib/Language/Drasil/Document.hs

```diff
@@ -87,7 +87,7 @@ checkToC (Notebook t a sc) = Notebook t a sc
 
 -- | Smart constructor for labelled content chunks.
 llcc :: Reference -> RawContent -> LabelledContent
-llcc = LblC
+llcc r = LblC (r ^. uid) r
 
```

Comment:
Note that here we are cloning the `UID` the inner-held `Reference` for its own `UID`. We can create new constructors for each LC type later, but I left this as-is for. In a subsequent PR, we can re-write the constructors for `L-C-`s so as to not "copy" the `UID` from the `Reference`, but to _copy its own `UID` to the `Reference`._

(I chose to not do that in this PR because my focus was on making #4489 happen, not cleaning up code related to `L-C-`s right now.)